build.sbt

```diff
@@ -99,5 +99,15 @@ lazy val example = project.module
     micrositeGitHostingUrl := "https://github.com/scalaz/scalaz",
     micrositeGithubOwner := "scalaz",
     micrositeGithubRepo := "scalaz",
-    micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512"))
+    micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512")),
+    micrositePalette := Map(
+      "brand-primary"   -> "#ED2124",
+      "brand-secondary" -> "#251605",
+      "brand-tertiary"  -> "#A62639",
```

Comment:
I think this doesn't contrast very well with the scalaz logo in the upper right.